workflow-templates/im-deploy-az-app-manually.yml

```diff
@@ -103,27 +110,71 @@ jobs:
           scope: ${{ env.ENVIRONMENT }}
           create-output-variables: true
         env:
-          # This variable is used to swap slots and annotate app insights
+          # This variable is used to swap slots
           RESOURCE_GROUP@dev: ''
           RESOURCE_GROUP@qa: ''
           RESOURCE_GROUP@stage: ''
+          RESOURCE_GROUP@stage-secondary: ''
           RESOURCE_GROUP@demo: ''
           RESOURCE_GROUP@uat: ''
           RESOURCE_GROUP@prod: ''
+          RESOURCE_GROUP@prod-secondary: ''
+
+          # Primary resource group used to pull azure credentials and secrets from
+          PRIMARY_RESOURCE_GROUP@dev: ''
+          PRIMARY_RESOURCE_GROUP@qa: ''
+          PRIMARY_RESOURCE_GROUP@stage stage-secondary: ''
+          PRIMARY_RESOURCE_GROUP@demo: ''
+          PRIMARY_RESOURCE_GROUP@prod prod-secondary: ''
```

Comment:
Unless I'm misunderstanding something, credentials ad secrets are not pulled from resource groups.

From reading the rest of the code, I guess the comment should be changed to say something more along the lines of `# Resource group holding the state storage account and managed service identities`

Reply:
Also, now that there are two variables holding resource group names, it might be better to rename the existing `RESOURCE_GROUP` variable to something like `TARGET_RESOURCE_GROUP` or `DESTINATION_RESOURCE_GROUP`

Reply:
Updated to TARGET_RESOURCE_GROUP and comment.